### PR TITLE
feat(auth): add topic tips

### PR DESCRIPTION
**Pull request by ysfscream**
@Kinplemelon tooltip component doesn't seem to have the width props.

### CODE REVIEW

File: src/i18n/Auth.js [hunk 1]
```diff
@@ -303,4 +303,8 @@ export default {
     zh: '写模式',
     en: 'Write Mode',
   },
+  topicTips: {
+    zh: '主题字符串，支持 ${0} 和 ${1} 占位符，在执行权限检查时会被替换为对应客户端的 Username 或 Client ID',
```
Comment (Kinplemelon) [inline src/i18n/Auth.js:307]: can use "主题字符串，支持 {'{'}username{'}'} 和 {'{'}clientid{'}'} 占位符..."?
like this https://vue-i18n.intlify.dev/guide/essentials/syntax.html#literal-interpolation

Comment (Kinplemelon) [reply src/i18n/Auth.js:307]: > @Kinplemelon tooltip component doesn't seem to have the width props.

oh, should I add width prop?

Comment (ysfscream) [reply src/i18n/Auth.js:307]: Oh, thanks so much. I've been looking all morning and haven't found out how to fix it. 😂

Comment (ysfscream) [reply src/i18n/Auth.js:307]: > > @Kinplemelon tooltip component doesn't seem to have the width props.
> 
> oh, should I add width prop?

YES, it's useful.

**Pull request merged**